src/doc/unstable-book/src/abi-ptx.md

```diff
@@ -1,5 +1,59 @@
 # `abi_ptx`
 
-The tracking issue for this feature is: None.
+The tracking issue for this feature
+is: [38788](https://github.com/rust-lang/rust/issues/38788)
```

Comment:
we have a standard form for this stuff, like this: https://github.com/rust-lang/rust/pull/41135/files#diff-d6ab7d3e772d67c4944af27d29e229cdR3

mind changing this to be identical? Thanks 😄